### ISSUE TITLE
Install commercial version of Burp Suite on Kali

### ISSUE THREAD

**Issue by jsf9k** [cisagov/cool-system#34]
# 🚀 Feature Proposal

Install the commercial version of Burp Suite on the Kali AMI.

## Motivation

The assessment folks have asked for this.

**Comment by jsf9k** [comment]
Assuming everyone uses a shared key like we do with Cobalt Strike (we buy sufficient licenses, but they all use the same key) [it looks like I can do this at AMI creation time](https://gist.github.com/pajswigger/f0caac124a02d94aa1ebbc46921d84ea
) via [the expect module in Ansible](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/expect_module.html).

@justinforbes, can you help me obtain the license key for Burp Pro?  Can you also verify my assumption that the licensing works just like the Cobalt Strike licensing?

Credit where credit is due:  I believe @mcdonnnj provided me with that Gist showing how to license Burp via `expect`.

**Comment by justinforbes** [comment]
Yes burp and cobalt strike are per user licenses so we should be good as long as we have more licenses than users.

**Comment by jsf9k** [comment]
Great!  Can you send me a Burp Pro license key through email or chat?  Or tell me who to ask?

**Comment by justinforbes** [comment]
I go through Brian Chesney for burp licenses, the one I have is about to expire.  I think he was working on a license inventory.

**Comment by jsf9k** [comment]
> I go through Brian Chesney for burp licenses, the one I have is about to expire. I think he was working on a license inventory.

Thanks, I will ask him.

**Comment by jsf9k** [comment]
@justinforbes, I have a license now.  Do you know if Burp Suite Pro requires a special installer, or if I can just download the free one and apply the license to it?  For Cobalt Strike there is a non-free tarball that was given to me.  I store it somewhere safe and pull it in when I install.

**Comment by justinforbes** [comment]
It does have a separate installer from the free version, whomever controls the burp licenses should be able to log in and download the installer.

**Comment by jsf9k** [comment]
> It does have a separate installer from the free version, whomever controls the burp licenses should be able to log in and download the installer.

Thanks, I will ask Brian again.

**Comment by jsf9k** [comment]
I now have the license and the installers.
